### ISSUE TITLE
Doesn't work anymore (OSX)

### ISSUE THREAD

**Issue by essenciary** [KristofferC/PimpMyREPL.jl#13]
Not sure what happened - it used to work upon installing it, was super cool! 
Now it doesn't do anything :( 

Using iTerm on OSX, but it's the same with Terminal. 

```julia
julia> versioninfo()
Julia Version 0.4.6
Commit 2e358ce* (2016-06-19 17:16 UTC)
Platform Info:
  System: Darwin (x86_64-apple-darwin15.5.0)
  CPU: Intel(R) Core(TM) i7-3740QM CPU @ 2.70GHz
  WORD_SIZE: 64
  BLAS: libopenblas (DYNAMIC_ARCH NO_AFFINITY Sandybridge)
  LAPACK: libopenblas
  LIBM: libopenlibm
  LLVM: libLLVM-3.3
```

**Comment by KristofferC** [comment]
Could you please try to run `Pkg.update()` and see if things work again?

![image](https://cloud.githubusercontent.com/assets/1282691/17887299/1ba55432-6926-11e6-8f0c-9b1c4b7a8ac6.png)

**Comment by essenciary** [comment]
Thanks, same problem, please see screenshot (I have PimpMyREPL loaded automatically in .juliarc.jl)
https://monosnap.com/file/9QVgYahzfVvgfD5HcBnD2H5bPTYJKW

**Comment by KristofferC** [comment]
Try to manually run `PimpMyREPL.Prompt.hijack_REPL()`.

**Comment by KristofferC** [comment]
There have been some updates now. Could you try it again to see if there is still a problem?

**Comment by okvs** [comment]
I've been updating PimpMyREPL several times the last few minutes. I saw the same problem, `using PimpMyREPL` was called without errors, but it did nothing (no highlighting etc.)

Running `PimpMyREPL.Prompt.hijack_REPL()` _did_ work, but enabling bracket matching did _not_ work. I tried that a few times, to no avail.

Then I decided to restart Julia, but now it won't start. The terminal just closes after calling `$ exec /.../julia`

I'm running the RC3 binary on osx 10.11.6.

**Comment by okvs** [comment]
I'd actually like to just temporarily remove PimpMyREPL to see it that's the problem, but without access to terminal I'm a bit uncertain how to go about it. Can I just delete some folder somewhere?

**Comment by KristofferC** [comment]
You can stop PimpMyREPL from start automatically by just removing the `setup()` call in your `.juliarc`.

**Comment by KristofferC** [comment]
You can also remove the `.julia/v0.x/PimpMyREPL` folder.

**Comment by okvs** [comment]
[slaps forehead] That works, of course. And now, `using PimpMyREPL` works again, so there's something wrong with the code in `.juliarc` that I need to figure out.

**Comment by KristofferC** [comment]
Try to tweak the magic numbers there a bit (increase them).

**Comment by KristofferC** [comment]
I think I have a much more reliable way of doing this now. Will make a PR and maybe you can try it for me.

**Comment by okvs** [comment]
Actually, it seems like these two lines are causing the problems:
```
import PimpMyREPL.Passes.SyntaxHighlighter: ColorScheme, SYNTAX_HIGHLIGHTER_SETTINGS, add!
import PimpMyREPL.ANSICodes.ANSIToken
```
Whenever I comment them out everything is fine, when i uncomment, Julia refuses to start.

**Comment by KristofferC** [comment]
I see, and I think I know the problem. Working on it right now.

**Comment by KristofferC** [comment]
This should be fixed by #19 to not be racy. Please comment if it is not. Now you only need to put `using PimpMyREPL` in the `.juliarc.jl`
